code/modules/reagents/reagent_containers/food/snacks/cultural/human.dm

```diff
@@ -417,3 +462,115 @@
 	reagent_data = list(/singleton/reagent/nutriment = list("blue raspberry" = 5, "white chocolate" = 3))
 	drop_sound = 'sound/items/drop/glass.ogg'
 	pickup_sound = 'sound/items/pickup/glass.ogg'
+
+
+// Eridani
+
+/obj/item/reagent_containers/food/snacks/bowl
+	name = "a bowl of item"
+	desc = "If you're seeing this, something has gone wrong D:"
+	icon = 'icons/obj/item/reagent_containers/food/cultural/human.dmi'
+	icon_state = "puffpuffbowl"
+	trash = /obj/item/trash/snack_bowl
+	var/vendingobject = /obj/item/reagent_containers/food/snacks/puffpuff
+	reagent_data = list(/singleton/reagent/nutriment = list("fried dough" = 10, "ginger" = 4))
+	bitesize = 4
+	reagents_to_add = list(/singleton/reagent/nutriment = 24)
+	var/unitname = "item"
```

Comment:
DMdoc this

Reply:
I think I did it? If I understood correctly. Please check if I did what you needed me to do. Thank you!